docs/source/API/core/initialize_finalize/InitArguments.rst

```diff
@@ -0,0 +1,64 @@
+InitArguments
+=============
+
+.. role:: cppkokkos(code)
+   :language: cppkokkos
+
+Defined in ``<Kokkos_Core.hpp>`` header.
+
+.. warning:: Deprecated since 3.7
+
+Interface
+---------
+
+.. cppkokkos:struct:: InitArguments
+
+   .. cppkokkos:member:: int num_threads
+
+   .. cppkokkos:member:: int num_numa
+
+   .. cppkokkos:member:: int device_id
+
+   .. cppkokkos:member:: int ndevices
+
+   .. cppkokkos:member:: int skip_device
+
+   .. cppkokkos:member:: bool disable_warnings
+
+   .. cppkokkos:function:: InitArguments()
+
+.. _KokkosInitialize: initialize.html
+.. |KokkosInitialize| replace:: ``Kokkos::initialize``
+
+.. _KokkosInitializationSetting: InitializationSettings.html
+.. |KokkosInitializationSetting| replace:: ``Kokkos::InitializationSettings``
```

Comment:
I would put these link at the top of the page so that we can reuse them a few times and are not in the way here in the middle of the page